mets.md

```diff
@@ -160,6 +199,83 @@ encodings of the same page.
 </mets:structMap>
 ```
 
+## OCR-D structMap
+
+A METS can contain different `structMap`. These are defined by the `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL`>) described in more detail.	
+A document structure determined by OCR-D is described by `TYPE="OCR-D-LOGICAL"`.
+
+attributes in `structMap` | description
+--                        | --
+`LABEL`                   | contains the character recognition result  e.g. the title of the chapter
+`TYPE`                    | contains a standardized description of structur (see.: [DFG-Viewer: structural data set](https://dfg-viewer.de/strukturdatenset/))
+
+### `mets:structLink`
+
+The `mets:structLink`describes the range of pages in part of document.
+
+
+### Example
+
+```xml
+<mets:fileGrp USE="OCR-D-IMG">
+    <mets:file ID="OCR-D-IMG_0001" >...</mets:file>
+</mets:fileGrp>
+<mets:fileGrp USE="OCR-D-OCR">
+    <mets:file ID="OCR-D-OCR_0001" >...</mets:file>
+</mets:fileGrp>
+<mets:structMap TYPE="OCR-D-LOGICAL">
+  <mets:div DMDID="dmdSec_0001" ADMID="amdSec_0001" ID="OCR-D-loc_0001">
+    <mets:div ID="OCR-D-loc_d5e320" TYPE="chapter" LABEL="Kapidel 1">
```

Comment:
Is this a typo or an intentional "recognition error"?

Reply:
This is a simulated "recognition error".

Reply:
I see. Then maybe this would be more realistic / clear:
```suggestion
    <mets:div ID="OCR-D-loc_d5e320" TYPE="chapter" LABEL="KapıteI 1">
```